_posts/2017-01-16-Jak-psat-php-Uloziste-kodu+databaze+frameworky.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Jak psát PHP? (3/3) - Úložiště kódu, databáze a frameworky
+---
+
+V poslední části přehledu o nástrojích vhodných pro jazyk PHP se podíváme nejen na github + gitlab, řekneme si něco málo o databázích a předestřu základy frameworků. Tady můžete najít [druhý](http://jakpsatphp.cz/Jak-psat-php-Zavislosti+verzovaci-systemy+debugging/) nebo [první](http://jakpsatphp.cz/Jak-psat-php-Zdroje-znalosti+vyvojove-prostredi/) díl série. 
+
+# Úložiště kódu
+Když nebudete pracovat sami nebo se budete chtít pochlubit novým projektem,
+ 
+Všechny úložiště poskytují základní funkce jako jsou větvení a klonování kódu, Pull Requesty, Code Review, Issues nebo Markdown Support. 
```

Comment:
Toto bych asi hodil za jednotlivá úložiště, protože tady ještě nevím, co to je. Ale už mám rozumět tomu, že všechny (něco) poskytují (další nové pojmy).